app/ui.js

```diff
@@ -1053,18 +1053,17 @@ var UI;
             }
         },
 
-        connect: function() {
+        connect: function(_, password) {
```

Comment:
When `connect()` is called via the connect button, the first parameter is the DOM element that was clicked on. We don't need that, so it is ignored.